evap/results/exporters.py

```diff
@@ -101,11 +106,18 @@ def filter_text_and_heading_questions(questions):
         return filtered_questions
 
     @staticmethod
-    def filter_evaluations(semesters, evaluation_states, degrees, course_types, contributor, include_not_enough_voters):
+    def filter_evaluations(
+        semesters: Iterable[Semester],
+        evaluation_states: Iterable[int],
+        degrees: Iterable[int],
+        course_types: CourseType,
```

Comment:
This is wrong, course_types is an Iterable (the code here should work for `Union[CourseType, int]`, but it seems like we only ever pass `int`s)